### PR TITLE
ACS-8729: Increased readability of statusCode verification

### PR DESCRIPTION
**Pull request by krdabrowski**

### CODE REVIEW

**Review by tpage-alfresco — APPROVED** [review]
LGTM - thanks!

File: common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/AcsHealthProbe.java [hunk 1]
```diff
@@ -103,4 +98,9 @@ private void sleep() throws InterruptedException
         log.debug("ACS is not available. Retrying in {} seconds", retryIntervalSeconds);
         TimeUnit.SECONDS.sleep(retryIntervalSeconds);
     }
+
+    private static boolean isErrorCode(int statusCode)
```
Comment (mpichura) [inline common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/AcsHealthProbe.java:102]: Maybe this method could be something like `isNotErrorCode` and we could avoid negating it when it's called.

**Review by mpichura — COMMENTED** [review]
See my small suggestion

File: common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/AcsHealthProbe.java (hunk 1 above)
Comment (krdabrowski) [reply common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/AcsHealthProbe.java]: Fixed

**Pull request merged**